src/assets/Keycloak/keycloak.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Introduction to Keycloak
+authorName: Amrita Shrestha
+authorAvatar: https://avatars.githubusercontent.com/u/54478846?s=100&v=4
+authorLink: https://github.com/amrita-shrestha
+createdAt: March 26, 2024
+tags: Keycloak
+banner: https://blog.jankaritech.com/src/assets/keycloak/images/keycloak.png
+---
+
+## Keycloak
+[Keycloak](https://github.com/keycloak/keycloak) is an open-source project for Red Hat SSO. It provides an `Identity and Access Management` solutions designed to secure application services.
+Additionally, it enables users to authenticate through various identity providers and use fine-grained permissions for regulating access to Software as a Service (SaaS) applications.
+Keycloak alternatives are Auth0, Zluri, Microsoft Azure Active Directory, Okta or WSO2 Identity Server.
```

Comment:
here you are talking currently about what keycloak can do, then suddenly about alternatives and again about what it can do. maybe shift the alternatives to the end of the chapter